de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/v1_9/fsk2metadata/FSK2MetadataNodeModel.java

```diff
@@ -114,10 +116,14 @@ private static DataCell createJSONCell(final Object object) {
         new DataColumnSpecCreator("dataBackground", JSONCell.TYPE).createSpec();
     final DataColumnSpec modelMathSpec =
         new DataColumnSpecCreator("modelMath", JSONCell.TYPE).createSpec();
+    final DataColumnSpec scriptSpec =
+        new DataColumnSpecCreator("modelscript", JSONCell.TYPE).createSpec();

```

Comment:
Should the model and visualisation scripts be contained in JSON cells or String cells?

Reply:
good catch!